lib/papertrail/cli.rb

```diff
@@ -95,6 +95,9 @@ def run
 
       @connection = Papertrail::Connection.new(options)
 
+      # Use HTTP Keep-Alive unless delay is too long
+      connection.start if options[:delay] < 10
```

Comment:
I think this will crash if `options[:delay]` is nil - not seeing where a default is set?

Reply:
ooo ignore that `nil.to_i == 0`